logic/src/commonMain/kotlin/com/wire/kalium/logic/data/client/ClientRepository.kt

```diff
@@ -253,6 +253,7 @@ class ClientDataSource(
         newClientEvent: Event.User.NewClient
     ): Either<CoreFailure, Unit> = wrapStorageRequest {
         newClientDAO.insertNewClient(clientMapper.toInsertClientParam(selfUserID, newClientEvent))
+        clientDAO.insertClient(clientMapper.toInsertClientParam(selfUserID, newClientEvent))
```

Comment:
What's the different between these two lines?

Reply:
different DB-tables where we save clients into. 
`NewClient` table containse only self-user clients that was added and user wasn't informed about it yet. So it's used for informing user about new devices by the dialog (kinda "there is a new device, was it you?"). 

And the `Client` table - is a table for all the clients